### PR TITLE
add migration readiness report

### PR DESCRIPTION
**Pull request by AlexicaWright**
Images to come!

### CODE REVIEW

File: modules/ROOT/pages/managing-instances/migration-readiness.adoc [hunk 1]
```diff
@@ -0,0 +1,102 @@
+= Migration Readiness Report
+:description: This section describes migration readiness report for upgrading from aura 4 to latest.
+:page-aliases: migration-readiness.adoc
+
+[NOTE]
+====
+The Migration Readiness Report is only available for Instances on Neo4j version 4.
+You can access the report for any version 4 instance via the link through the "Migration recommended" label on the Instance and Console pages.
+====
+
```
Comment (SaschaPeukert) [inline modules/ROOT/pages/managing-instances/migration-readiness.adoc:10]:
```suggestion

[NOTE]
====
In the Migration Readiness Report we only highlight issues that need to be changed to make your code and queries compatible with AuraDB v5.
While you are working on this, we would also encourage you to extend this work to take advantage of the new v5 features.
====

```

File: modules/ROOT/pages/managing-instances/migration-readiness.adoc [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+* The application name that executed the query (if set)
+* The initiation type of the query
+* The query text or parts of it
+
+Use the button in the popup to fetch applicable data to populate the report's table.
+
+image::mrr-deprecation-table.png[]
+
+Each row in the table represents a query in the selected timeframe that must be changed to seamlessly migrate to the latest Aura version.
+All executions of the same query are aggregated into one row (see also the "Count" column).
```
Comment (SaschaPeukert) [inline modules/ROOT/pages/managing-instances/migration-readiness.adoc:64]:
```suggestion
You have to rewrite those queries to only use v5 supported Cypher.

All executions of the same query are aggregated into one row (see also the "Count" column).
```

**Review by SaschaPeukert — COMMENTED** [review]
Feedback on the other PR from our PM

File: modules/ROOT/pages/managing-instances/migration-readiness.adoc (hunk 1 above)
Comment (AlexicaWright) [reply modules/ROOT/pages/managing-instances/migration-readiness.adoc:10]: This sounds too much like marketing and it is not good UX to add one admonition right after another. Let me see if I can somehow convey this information in a different way.

File: modules/ROOT/pages/managing-instances/migration-readiness.adoc [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+* Cypher deprecations
+* Deprecated driver usage
+* Deprecated index types
+
+The report page has a section for each category and a chart at the top titled "Deprecations and query timeline."
+This document explains each section, but the first is vital to controlling the others and is detailed in the following section.
+
+Note that the report only highlights issues that need to be addressed to make your code and queries compatible with AuraDB v5.
+Working on these issues presents a good opportunity to learn more about the advanatges of the new features in version 5.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/managing-instances/migration-readiness.adoc:27]:
```suggestion
Note that the report only highlights issues that need to be addressed to make your code and queries compatible with AuraDB v5.
Working on these issues presents a good opportunity to learn more about the advantages of the new features in version 5.
```

File: modules/ROOT/pages/managing-instances/migration-readiness.adoc (hunk 1 above)
Comment (AlexicaWright) [reply modules/ROOT/pages/managing-instances/migration-readiness.adoc:10]: I've added a paragraph about this instead of in the admonition.

**Review by SaschaPeukert — APPROVED** [review]

**Pull request merged**